rules/common.smk

```diff
@@ -1,20 +1,23 @@
 import pandas as pd
+from peppy import Project
 from snakemake.utils import validate
 from snakemake.utils import min_version
 
 min_version("5.7.1")
 
 report: "../report/workflow.rst"
 
+p = Project(cfg="prjcfg.yaml", sample_table_index="sample",
+            subsample_table_index=["sample", "unit"])
+
```

Comment:
`sample_table_index` and `subsample_table_index` arguments could be skipped if snakemake followed `peppy` defaults: 

- `"sample_name"` for sample_table index colnames
- `["sample_name", "subsample_name"]` for subsample_table index colnames